client/src/pages/Configurations/ConfigActivate/modals/SwitchActivationModal.tsx

```diff
@@ -0,0 +1,51 @@
+import { ModalRef, ModalToggleButton } from '@trussworks/react-uswds';
+import { GenericModal } from './GenericModal';
+import { Button, SECONDARY_BUTTON_STYLES } from '../../../../components/Button';
+import { RefObject } from 'react';
+
+interface SwitchActivationModal {
+  curVersion: number;
+  activeVersion: number | null;
+  handleActivation: () => void;
+  modalRef: RefObject<ModalRef | null>;
+}
+export function SwitchActivationModal({
+  modalRef,
+  curVersion,
+  activeVersion,
+  handleActivation,
+}: SwitchActivationModal) {
+  return (
+    <GenericModal
+      modalRef={modalRef}
+      title={`Switch to Version ${curVersion}`}
+      body={
+        <div>
+          <p id="activation-confirmation-modal-text" className="my-6">
+            You're about to stop Version {activeVersion} and start Version{' '}
+            {curVersion}
+          </p>
+          <p>
+            The eCR pipeline will begin using Version {curVersion}{' '}
+            <b>immediately</b>
```

Comment:
Extremely minor but I'd probably lean towards this instead of using a `<b>` tag here since [that tag has a semantic meaning](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/b) that doesn't really apply in this case.
```suggestion
            <span className="font-bold">immediately</span>
```